infrastructure/datastore/db.go

```diff
@@ -0,0 +1,71 @@
+package datastore
+
+import (
+	"bufio"
+	"encoding/csv"
+	"io"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/wmaldonadoc/academy-go-q42021/config"
+	"github.com/wmaldonadoc/academy-go-q42021/domain/model"
+	"go.uber.org/zap"
+)
+
+func openFile() [][]string {
+	fileLocation := config.GetEnvVariable("FILE_LOCATION")
+	file, err := os.Open(fileLocation)
+	if err != nil {
+		zap.S().Errorf("Error opening the CSV file %s", err)
+
+	}
+	zap.S().Info("Successfully openned csv file")
+
+	defer file.Close()
+
+	row1, err := bufio.NewReader(file).ReadSlice('\n')
+	if err != nil {
+		zap.S().Errorf("Error reading the CSV file %s", err)
+		log.Fatal(err)
+	}
+	_, err = file.Seek(int64(len(row1)), io.SeekStart)
+	if err != nil {
+		zap.S().Errorf("Error reading the CSV file %s", err)
+		log.Fatal(err)
+	}
+
+	chunks, err := csv.NewReader(file).ReadAll()
+	if err != nil {
+		zap.S().Errorf("Error reading the CSV file %s", err)
+		log.Fatal(err)
+	}
+
+	return chunks
+}
+
+func NewDb() []*model.Pokemon {
```

Comment:
Technically speaking, a .`csv file` is not a `DB`, so I'd recommend having a proper name for this method. Let's say `NewCSV` or something like that.

Also, as a rule of thumb, every exported func, var, type, etc. definition should have a comment above it to briefly explain what it does. These comments are used as a kind of documentation for you and other people that might be using your code. Let's encourage this kind of practice from now on in each of your project's files.

```suggestion
// NewCSV reads a .csv file and returns an array with all the pokemons found in this file
func NewCSV() []*model.Pokemon {
```